script/1_0_2current_en_script/Arcueid/Day 5/05_00_ARC05_5B.txt

```diff
@@ -1,31 +1,31 @@
 <Page1>
-It was just like a human missile.
+She was just like a human missile.
 Arcueid descended from a height of 80 meters above Vlov.
 <Page2>
 No, it was more like she flew.
 While in the air, she took aim and angled herself before accelerating like a missile.
 <Page3>
-Sand and debris billow up into the air.
-The wind whipped up a cloud of dust that sweeps the area.
-The river of flame that had swept across the downtown area was eradicated by that single blow.
+Earth and debris billow up into the air.
+The wind whipped up a cloud of dust that scoured the area.
+The river of flame that had been flowing over the downtown area was halted by that single blow.
 <Page4>
-The policemen that surround Vlov in the distance are blown away, incapacitating most of them. They are divided into two groups: those who became unconscious after suffering a head injury and those who lose their will to fight.
+The policemen that had been encircling Vlov are blown away, and most of them are incapacitated in the process. Half of them have been knocked unconscious by blows to the head, and the rest have simply lost their will to fight.
 It's not surprising.
 <Page5>
 "In-Incre..."
-After the dust had settled, the result was in plain view.//to be rechecked "this is the result" sounds bad apparently, なにしろ
-Arcueid's rapid descent not only shattered the area around Vlov, but also a large portion of the downtown area at the north gate.
+Once the dust has settled, the extent of the destruction is plain to see. // これである reified for flow
```

Comment:
had
(the destruction can only be seen once the dust has finished settling)

Reply:
On this and the next one, is that definitely the correct tense? I assume he is talking about things in the moment, so if something happened just a second ago I feel like `has` is correct? Would `had` not be if he were recounting this story at a later date?

Reply:
Tried reading it several times but I still stand by my suggestion. Will have to wait for a third opinion

Reply:
On this and the next one, is that definitely the correct tense? I assume he is talking about things in the moment, so if something happened just a second ago I feel like `has` is correct? Would `had` not be if he were recounting this story at a later date?

Reply:
I agree with Ross but propose "Once the dust settles"